t/ui/01-list.t

```diff
@@ -57,6 +57,13 @@ my %job_param = (
 # We do not need job 99981 right now so delete it here just to have a helpful
 # example for customizing the test database
 sub prepare_database {
+    my $bugs     = $schema->resultset('Bugs');
+    my %bug_args = (refreshed => 1, existing => 1);
```

Comment:
Looks like this isn't really used as a hash so it would be possible to use an array:

```suggestion
    my @bug_args = (refreshed => 1, existing => 1);
```
(Not sure what's generally preferred here.)

Reply:
Hm, I'm not sure why you think it isn't used as a hash...

Reply:
It isn't used like `$bug_args{some_key}`, just expanded in list context.

Reply:
Ah, I see.
I don't see a reason to use an array though.
But if at least one more person suggests that, I'll change it :)

Reply:
Leave it imho